Makefile

```diff
@@ -19,7 +19,12 @@ MIGRATIONS:=$(shell find db/migrations -type f)
 all: $(GOPATH)/bin/fluxctl $(GOPATH)/bin/fluxd $(GOPATH)/bin/fluxsvc build/.fluxd.done build/.fluxsvc.done
 
 .PHONY: release-bins
-release-bins: build/fluxctl-linux-amd64 build/fluxctl-darwin-amd64
+release-bins:
```

Comment:
Nope; if it did, then `make release-bins` would not always build binaries with the current version baked in (sometimes it would look at the dependencies and decide it didn't need to).